fmriprep/cli/parser.py

```diff
@@ -150,6 +150,7 @@ def _bids_filter(value):
         "--nthreads",
         "--n_cpus",
         "--n-cpus",
+        dest='nprocs',
```

Comment:
why is not ``--nprocs`` used as `dest`?

Reply:
i think it'd be nice to have it explicitly said, especially since this changed from `20.0.x`

Reply:
> explicitly said

Well, developers know that argparse maps `--nprocs` to `parsed.nprocs` if it is placed first (which is implicit, but clear in their API). To the user, I don't think this makes any difference. Should we define `dest` for all arguments then?

Reply:
if they all had multiple aliases like this one, i tend to side with "yes"

Reply:
@oesteban 👍?